tool/travis.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Created with https://github.com/dart-lang/mono_repo
+
+# Fast fail the script on failures.
+set -e
+
+if [ -z "$PKG" ]; then
+  echo -e "[31mPKG environment variable must be set![0m"
+  exit 1
+elif [ -z "$TASK" ]; then
+  echo -e "[31mTASK environment variable must be set![0m"
+  exit 1
+fi
+
+pushd $PKG
+pub upgrade
+
+case $TASK in
+dartanalyzer) echo
```

Comment:
wow, I have never used a case statement in a shell script before......... this syntax is insane lol